src/components/TagList/TagList.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="tags">
+    <i v-if="icon" class="pi" :class="icon" />
+    <Tag flat>
+      {{ internalTags }}
+      <template v-if="tagLengthLimit">
+        <Tooltip  placement="bottom">
+          <template v-slot:trigger>
+            <span class="tags__showTooltip">show</span>
+          </template>
+          <span class="tags__tooltipTag">{{ jointTags }}</span>
+        </Tooltip>
+      </template>
+    </Tag>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import Tag from "@/components/Tag/Tag.vue";
+import Tooltip from "@/components/Tooltip/Tooltip.vue"
+
+export default defineComponent({
+  name: "TagList",
+  components: { Tag, Tooltip },
+  props: {
+    icon: {
+      type: String,
+      default: null
+    },
+    tags: {
+      type: Array,
+      default: function () { return [] }
+    },
+    limit: {
+      type: Number,
+      default: 30
```

Comment:
Took me a little bit to figure out this is a character limit and not a tag limit. I expected this to display 30 tags before truncating. I'm curious what other's thoughts are on this. @stackoverfloweth @znicholasbrown

Reply:
I can rename it to `characterLimit`

Reply:
After talking with @joeclayallday a little I think we want to focus on number of tags shown.